test/typescript/context.test.tsx

```diff
@@ -1,4 +1,36 @@
-import i18next from 'i18next';
+import i18n from './i18n';
```

Comment:
This import is incorrect.  You are importing a local file, instead of the module, which would disable module augmentation.  This should be switched back.